app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="current-user--row">
+    <thumbnail
+      :src="currentUser.avatar_url"
+      :username="currentUserAvailableName"
+    />
+    <div class="current-user--data">
+      <h3 class="current-user--name">
+        {{ currentUserAvailableName }}
+      </h3>
+      <h5 v-if="currentRole" class="current-user--role">
+        {{ $t(`AGENT_MGMT.AGENT_TYPES.${currentRole.toUpperCase()}`) }}
+      </h5>
+    </div>
+  </div>
+</template>
+<script>
+import { mapGetters } from 'vuex';
+import Thumbnail from '../../widgets/Thumbnail';
+
+export default {
+  components: {
+    Thumbnail,
+  },
+  computed: {
+    ...mapGetters({
+      currentUser: 'getCurrentUser',
+      currentRole: 'getCurrentRole',
+    }),
+    currentUserAvailableName() {
+      return this.currentUser.name;
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.current-user--row {
+  align-items: center;
+  display: flex;
+}
+
+.current-user--data {
+  display: flex;
+  flex-direction: column;
+
+  .current-user--name {
+    font-size: var(--font-size-small);
+    font-weight: var(--font-weight-medium);
+    line-height: 1;
```

Comment:
Should have a case for overflow else lineheight 1 will look weird

Reply:
Removed the line-height. I don't think we need to show the entire name in the sidebar. I added text-truncate to the name.